pycbc/events/stat.py

```diff
@@ -878,6 +876,9 @@ def single(self, trigs):
         stat = thresh - (logr_n / self.alpharef)
         return numpy.array(stat, ndmin=1, dtype=numpy.float32)
 
+    def single_multiifo(self, s):
+        return s[1]['snglstat']
```

Comment:
Note that I should probably use the self.single_increasing part of https://github.com/gwastro/pycbc/pull/3345 here